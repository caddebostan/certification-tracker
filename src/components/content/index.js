import Content from './view/contentView';

export { Content };
