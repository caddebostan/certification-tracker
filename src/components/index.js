// ##############################
// // // Cards
// #############################

import ChartCard from './cards/chartCard';
import ProfileCard from './cards/profileCard';
import RegularCard from './cards/regularCard';
import StatsCard from './cards/statsCard';
import TasksCard from './cards/tasksCard';

// ##############################
// // // CustomButtons
// #############################

import Button from './customButtons/button';
import IconButton from './customButtons/iconButton';

// ##############################
// // // CustomInput
// #############################

import CustomInput from './customInput/customInput';

// ##############################
// // // Footer
// #############################

import Footer from './footer/footer';

// ##############################
// // // Grid
// #############################

import ItemGrid from './grid/itemGrid';

// ##############################
// // // Header
// #############################

import Header from './header/header';
import HeaderLinks from './header/headerLinks';

// ##############################
// // // Sidebar
// #############################

import Sidebar from './sidebar/sidebar';

// ##############################
// // // Snackbar
// #############################

import Snackbar from './snackbar/snackbar';
import SnackbarContent from './snackbar/snackbarContent';

// ##############################
// // // Table
// #############################

import Table from './table/table';

// ##############################
// // // Tasks
// #############################

import Tasks from './tasks/tasks';

// ##############################
// // // Typography
// #############################

import P from './typography/p';
import Quote from './typography/quote';
import Muted from './typography/muted';
import Primary from './typography/primary';
import Info from './typography/info';
import Success from './typography/success';
import Warning from './typography/warning';
import Danger from './typography/danger';
import Small from './typography/small';
import A from './typography/a';

export {
  // Cards
  ChartCard,
  ProfileCard,
  RegularCard,
  StatsCard,
  TasksCard,
  // CustomButtons
  Button,
  IconButton,
  // CustomInput
  CustomInput,
  // Footer
  Footer,
  // Grid
  ItemGrid,
  // Header
  Header,
  HeaderLinks,
  // Sidebar
  Sidebar,
  //Snackbar
  Snackbar,
  SnackbarContent,
  // Table
  Table,
  // Tasks
  Tasks,
  // Typography
  P,
  Quote,
  Muted,
  Primary,
  Info,
  Success,
  Warning,
  Danger,
  Small,
  A,
};
