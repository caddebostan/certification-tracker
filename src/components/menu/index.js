import Menu from './view/menuView';

export { Menu };
