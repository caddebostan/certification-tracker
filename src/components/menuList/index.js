import MenuListItems from './view/menuListView';

export default { MenuListItems };
