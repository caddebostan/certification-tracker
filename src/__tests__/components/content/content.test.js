import React from 'react';
import { shallow } from 'enzyme';

import { CustomInput } from '../../../components';

it('renders correctly', () => {
  const wrapper = shallow(<CustomInput />);

  expect(wrapper).toMatchSnapshot();
});
