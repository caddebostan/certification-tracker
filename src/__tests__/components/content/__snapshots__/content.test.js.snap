// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<WithStyles(ResponsiveDrawer)>
  <main
    className="content"
  />
</WithStyles(ResponsiveDrawer)>
`;
